src/routes/api/index.js

```diff
@@ -1,8 +1,9 @@
 import express from 'express';
 import passport from 'passport';
+import userRoutes from './users';
 
 const usersRouter = express.Router();
-
+usersRouter.use('/', userRoutes);
 usersRouter.post('/users/login', (req, res, next) => {
   if (!req.body.user.email) {
```

Comment:
@sebaztyn We should try to avoid write middlewares or controller code inside the route. This validation should be done in a middleware and used here

Reply:
@Stylll , thank you for the feedback. I have taken off the code. That was a code that came with the cloned codebase.